echo/echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/SlackNotificationAgent.groovy

```diff
@@ -101,15 +101,15 @@ class SlackNotificationAgent extends AbstractEventNotificationAgent {
 
     String address = preference.address
 
-    Response response
+    ResponseBody responseBody
     if (sendCompactMessages) {
-      response = slackService.sendCompactMessage(new CompactSlackMessage(body, color), address, true)
+      responseBody = slackService.sendCompactMessage(new CompactSlackMessage(body, color), address, true)
     } else {
       String title = getNotificationTitle(config.type, application, status)
-      response = slackService.sendMessage(new SlackAttachment(title, body, color), address, true)
+      responseBody = slackService.sendMessage(new SlackAttachment(title, body, color), address, true)
```

Comment:
Is there a test for this?

Reply:
GoogleChatClientTest has test for the similar case.

Reply:
ok